pack.toml

```diff
@@ -1,16 +1 @@
-name = ""
-author = "minMaximilian"
-version = "0.0.3"
-pack-format = "packwiz:1.1.0"
 
-[index]
-file = "index.toml"
-hash-format = "sha256"
-hash = "fafa6f01c8a03b2ba3e80931e29bd65ab75a759b4b7b07334ce47658ab76d4f2"
-
-[versions]
-fabric = "0.15.7"
-minecraft = "1.20.1"
-
-[options]
-acceptable-game-versions = []
```

Comment:
Nit:
Can you please not leave empty files if you are going to delete the contents of it?